tests/test_model.py

```diff
@@ -8,11 +8,13 @@
 from tempus_fugit_minecraft.utilities import WORLD_SIZE
 import random
 
+
 @pytest.fixture(scope="class")
 def model():
-    pyglet.options['audio'] = ('silent')
+    pyglet.options['audio'] = 'silent'
```

Comment:
Please undo this change.  From what I'm reading in the [pyglet options](https://pyglet.readthedocs.io/en/pyglet-1.5-maintenance/programming_guide/options.html) docs, I think we need this to be a tuple of just the one item since it says it should be a _sequence_.

Reply:
Oh okay, I thought the parenthesis were redundant so I removed them. Will revert.